python/chronos/src/bigdl/chronos/model/autoformer/Autoformer.py

```diff
@@ -151,19 +149,15 @@ def forward(self, x_enc, x_mark_enc, x_dec, x_mark_dec,
 
     def training_step(self, batch, batch_idx):
         batch_x, batch_y, batch_x_mark, batch_y_mark = map(lambda x: x.float(), batch)
-        dec_inp = torch.zeros_like(batch_y[:, -self.pred_len:, :]).float()
-        dec_inp = torch.cat([batch_y[:, :self.label_len, :], dec_inp], dim=1).float()
-        outputs = self(batch_x, batch_x_mark, dec_inp, batch_y_mark)
+        outputs = self(batch_x, batch_x_mark, batch_y, batch_y_mark)
 
         outputs = outputs[:, -self.pred_len:, :]
         batch_y = batch_y[:, -self.pred_len:, :]
         return self.loss(outputs, batch_y)
 
     def validation_step(self, batch, batch_idx):
         batch_x, batch_y, batch_x_mark, batch_y_mark = map(lambda x: x.float(), batch)
-        dec_inp = torch.zeros_like(batch_y[:, -self.pred_len:, :]).float()
-        dec_inp = torch.cat([batch_y[:, :self.label_len, :], dec_inp], dim=1).float()
-        outputs = self(batch_x.float(), batch_x_mark.float(), dec_inp,
+        outputs = self(batch_x.float(), batch_x_mark.float(), batch_y,
```

Comment:
do we still need these `.float()`?

Reply:
don't need anymore. I will remove these redundant code.